From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 31 Aug 2017 07:11:39 -0400
Subject: Skip test that started failing with 3.25

That way we can still benefit by running all the other tests.

Bug: https://bugzilla.gnome.org/785319
Applied-upstream: no
---
 clutter/tests/conform/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/clutter/tests/conform/Makefile.am b/clutter/tests/conform/Makefile.am
index 840e120..2bd7f4d 100644
--- a/clutter/tests/conform/Makefile.am
+++ b/clutter/tests/conform/Makefile.am
@@ -25,7 +25,6 @@ actor_tests = \
 	actor-iter \
 	actor-layout \
 	actor-meta \
-	actor-offscreen-redirect \
 	actor-paint-opacity \
 	actor-size \
 	$(NULL)
